src/patternfly/components/Dropdown/docs/code.md

```diff
@@ -40,3 +41,4 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-m-plain` | `.pf-c-dropdown__toggle` | Modifies to display the toggle with no border |
 | `.pf-m-active` | `.pf-c-dropdown__toggle` | Forces display of the active state of the element. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:active` pseudo-class. |
 | `.pf-m-disabled` | `a.pf-c-dropdown__menu-item` | Modifies to display the menu item as disabled. This applies to `a.pf-c-dropdown__menu-item` and should not be used in lieu of the `disabled` attribute on `button.pf-c-dropdown__menu-item`|
+| `.pf-m-disabled` | `div.pf-c-dropdown__toggle.pf-m-split-button` | Modifies to display the dropdown toggle as disabled. This applies to `div.pf-c-dropdown__toggle.pf-m-split-button` and should not be used in lieu of the `disabled` attribute on `button.pf-c-dropdown__toggle`, `.pf-c-dropdown__toggle-button`, or `.pf-c-dropdown__toggle-check > input[type="checkbox"]`|
```

Comment:
To me this seems like `.pf-m-disabled` can only apply to the dropdown with `.pf-m-split-button`. But can it be applied to every dropdown no matter what modifier is applied, if any modifier at all?

Reply:
You would only use it on `div.pf-c-dropdown__toggle` (not a button, you should use `disabled` instead), when you want the grey background color to appear - so not on a `.pf-m-plain` dropdown. Currently the only case for that in the dropdown is the split button toggle.

Reply:
Though this could probably just be written as `div.pf-c-dropdown__toggle`.

Reply:
^ I think that would be better